pytest_fixtures/core/broker.py

```diff
@@ -12,8 +12,8 @@
 
 def _resolve_deploy_args(args_dict):
     # TODO: https://github.com/rochacbruno/dynaconf/issues/690
-    args_dict = args_dict.copy().to_dict()
-    for key, val in args_dict.items():
+    args_dict_copy = args_dict.copy().to_dict()
+    for key, val in args_dict_copy.items():
```

Comment:
There shouldn't be a need to store the copy in this version. Additionally, to_dict may also not be needed.
```suggestion
    for key, val in args_dict.copy().to_dict().items():
```